scripts/appveyor.cmd

```diff
@@ -25,6 +25,7 @@ IF "%NAME%"=="python" (
     if !errorlevel! neq 0 exit /b !errorlevel!
     jlpm run build:test
     if !errorlevel! neq 0 exit /b !errorlevel!
+    setx FORCE_COLOR 1 && jlpm coverage:services --loglevel success

```

Comment:
I say we allow services to run twice since we inevitably just end up kicking it

Reply:
I think we already do run services twice (if it fails the first time) here:

https://github.com/jupyterlab/jupyterlab/blob/c4ec346ddf38887d876ddcdca8b5bc747967cbbd/tests/test-services/package.json#L8

I'm not actually sure if that's been working as intended, though

Reply:
Hmm, actually, there is one problem with how we're doubling up the run test-services cmd above: it messes up how command line arguments get passed down. For example, if I run:

```
setx FORCE_COLOR 1 && jlpm coverage:services --detectOpenHandles
```

it resolves to:

```
python run.py --coverage || python run.py --coverage --detectOpenHandles
```

I'll play around with this